index.html

```diff
@@ -72,15 +71,18 @@
                 <li class="dropdown">
                     <a class="dropdown-toggle" data-toggle="dropdown" href="#" >Documentation <span class="caret"></span></a>
                     <ul class="dropdown-menu">
-                        <li><a href="https://cwiki.apache.org/confluence/display/PINOT" target="_blank">Apache Wiki <i class="fa fa-external-link"></i></a></li>
-                        <li class="divider"></li>
                         <li class="dropdown-header">ReadTheDocs</li>
                         <li><a href="http://pinot.readthedocs.io/en/latest/" target="_blank">Latest <i class="fa fa-external-link"></i></a></li>
+                        <li class="divider"></li>
+                        <li><a href="https://cwiki.apache.org/confluence/display/PINOT" target="_blank">Wiki <i class="fa fa-external-link"></i></a></li>
+                        <li><a href="https://cwiki.apache.org/confluence/display/PINOT/Blogs+and+Talks" target="_blank">Blogs and Talks <i class="fa fa-external-link"></i></a></li>
```

Comment:
this might be worthwhile to make as a subpage instead of pointing to wiki -then community can open PR to add

Reply:
Reviewing a design doc on webpage/PR will be difficult. It's easier to comment on wiki.

Reply:
re design doc - not sure.. the projects I have seen working are JIRA/doc based.

talk links I think can be page.
https://raw.githubusercontent.com/apache/spark-website/asf-site/documentation.md
https://raw.githubusercontent.com/apache/spark-website/asf-site/third-party-projects.md
https://raw.githubusercontent.com/apache/spark-website/asf-site/powered-by.md

Reply:
Ok, I'll socialize this idea with the design doc owners who are working off of wiki as of now and will create a subsequent PR as required.